frameworks/helloworld/src/main/dist/examples/sidecar.yml

```diff
@@ -23,7 +23,9 @@ pods:
         cmd: "echo $TASK_NAME >> hello-container-path/output && sleep $SLEEP_DURATION"
         resource-set: hello-resources
         env:
-          SLEEP_DURATION: 1000
+          # Note, this is very high so that the test test_sidecar.test_toxic_sidecar_doesnt_trigger_recovery
+          # will always have an empty recovery plan.
+          SLEEP_DURATION: 10000
```

Comment:
After this https://github.com/mesosphere/dcos-commons/pull/1291 anything with a goal stat of FINISHED will never been in the recovery plan.